torch/csrc/jit/operator.h

```diff
@@ -81,6 +81,10 @@ struct TORCH_API Operator {
 };
 
 TORCH_API const std::vector<std::shared_ptr<Operator>>& getAllOperatorsFor(Symbol name);
+
+// returns a copy of the operator map
+TORCH_API const std::unordered_map<Symbol, std::vector<std::shared_ptr<Operator>>> getAllOperators();
```

Comment:
In particular, `getAllOperators()` makes me think that whatever you're going to do once you've gotten them all should probably just be a method on OperatorRegistry…since that's the thing we have that represents the set of all operators.